service-gateway-mock/nginx.conf

```diff
@@ -0,0 +1,26 @@
+# map specific LPA IDs to named examples in the Sirius Swagger file
+map $request_uri $swagger_example {
+    # 200 examples
+    "~A68582508781" 'registered';
+    "~A88668805824" 'rejected';
+    "~A93348314693" 'invalid';
+    "~A43476377885" 'withdrawn';
+
+    # 404 - results in a "waiting" status in Make
+    # NB there doesn't seem to be a way to supply an example for a non-200
+    # response so we just request an example which doesn't exist, which
+    # gives us a 404 response (albeit not a Sirius-styl 404 response)
+    "~A91155453023" 'NOTFOUND';
+
+    default         'received';
```

Comment:
I think we'd want a concrete `received` example instead, rather than defaulting 404 to `received`. My thinking is that 404 is probably more appropriate as the default (based on the comment above it), since this mimics the `notfound` real life scenario better. 🧵

Reply:
I think I did have it that way round at some point. Easy to change, so I'll do that.